### PR TITLE
Also limit the password length on reset

### PR DESCRIPTION
**Pull request by nickvergessen**
Ref https://github.com/nextcloud/server/pull/33110

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [x] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [x] Screenshots before/after for front-end changes
- [x] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [x] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by come-nc — APPROVED** [review]

**Review by PVince81 — APPROVED** [review]
:+1:

File: core/Controller/LostController.php [hunk 1]
```diff
@@ -240,6 +240,10 @@ public function setPassword(string $token, string $userId, string $password, boo
 			$this->eventDispatcher->dispatchTyped(new BeforePasswordResetEvent($user, $password));
 			\OC_Hook::emit('\OC\Core\LostPassword\Controller\LostController', 'pre_passwordReset', ['uid' => $userId, 'password' => $password]);
 
+			if (strlen($password) > 469) {
```
[the remark this answers is not in the document]
Comment (nickvergessen) [reply core/Controller/LostController.php:243]: Magic number for now due to upcoming release and necessary backport.
But yeah it yields for a const.

[the remark this answers is not in the document]
Comment (Aditya-Karbhari) [reply core/Controller/LostController.php:243]: Looks good to me.
Is the issue fixed or can I collabrate to fix this issue?

**Review by Aditya-Karbhari — COMMENTED** [review]
New change in code is fine to me.👍

File: core/Controller/LostController.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (nickvergessen) [reply core/Controller/LostController.php:243]: PR at https://github.com/nextcloud/server/pull/35981

**Pull request merged**